packit_service/worker/monitoring.py

```diff
@@ -0,0 +1,37 @@
+import logging
+import os
+
+from prometheus_client import CollectorRegistry, Counter, push_to_gateway
+
+logger = logging.getLogger(__name__)
+
+
+class Pushgateway:
+    def __init__(self):
+        self.pushgateway_address = (
+            "http://pushgateway" if os.getenv("DEPLOYMENT") in ("stg", "prod") else None
```

Comment:
I know this is going to be more complicated, but could we make this address to be configurable, defaulting to "http://pushgateway"?

With the current approach, I would need to change the code if I want to have this functionality on a local deployment, or I would need to name my deployment "stg" or "prod" which can be confusing.